public/javascripts/now.js

```diff
@@ -670,6 +672,46 @@ function updateWeather(coords, iconMap) {
   }, 600000);
 }
 
+/* Date and time functions */
+
+function initDatetime() {
+  updateDatetime();
+}
+
+function updateDatetime() {
+  $('#datetime #date').text(getCurrentDate());
+  $('#datetime #time').text(getCurrentTime());
+
+  setTimeout(function() {
+    updateDatetime();
+  }, 3000);
+}
+
+function getCurrentDate() {
+  var months = [
+    'January', 
+    'February', 
+    'March', 
+    'April', 
+    'May', 
+    'June', 
+    'July', 
+    'August', 
+    'September', 
+    'October', 
+    'November', 
+    'December'
+  ];
+  var date = new Date();
+  var month = months[date.getMonth()];
+  return month + ' ' + date.getDate() + ', ' + date.getFullYear();
+}
+
+function getCurrentTime() {
+  var date = new Date();
+  return date.getHours() + ':' + date.getMinutes();
```

Comment:
Pad the hours and minutes left with a zero for single digits.

Optional: This is a 24hr time. Would you consider adding a 12hr option, with configuration in `/app/config`? I'd like there to be a `Time Display` section at the top above `Weather Units` with options `12-hour` and `24-hour`. The 12hr option would display `AM` or `PM` as appropriate, without a leading zero for the hour, but the 24hr option would display a leading zero for a single digit hour. I'm willing to merge without this, but just give it a thought!